Intersect.Client.Core/Utilities/TextColorParser.cs

```diff
@@ -0,0 +1,69 @@
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace Intersect.Client.Utilities
+{
+    public static class TextColorParser
+    {
+        private static readonly Dictionary<string, Func<Color>> KnownColors = InitializeColorDictionary();
+        private static readonly Regex ColorTagRegex = new(@"\\c{([^}]*)}", RegexOptions.Compiled);
+        private static readonly string SplitPattern = @"(\\c{[^}]*})";
+
+        private static Dictionary<string, Func<Color>> InitializeColorDictionary()
```

Comment:
Can this actually be moved into `Color`, specifically the Color static constructor (`static Color()`) instead of a method called in an initializer.

Ideally `Color.FromString()` should be renamed to `Color.FromCsv()` and a new `Color.FromString()`  should be created